tests/ops/test_helpers/decorators.py

```diff
@@ -0,0 +1,29 @@
+import functools
+import os
+
+
+def override_envvars(env_vars):
```

Comment:
This is clever but see if you can make it work directly on the config. If this ends up being trickier than expected, you can use the existing pattern we've established for config overrides https://github.com/ethyca/fides/blob/LA-89-add-connection-retries-to-database-tasks/tests/fixtures/application_fixtures.py#L3228